app/routes/circulars/circulars.lib.ts

```diff
@@ -143,3 +145,63 @@ export function formatAuthor({
   else if (!affiliation) return `${name} <${email}>`
   else return `${name} at ${affiliation} <${email}>`
 }
+
+export function parseEventFromSubject(value: string): string {
```

Comment:
TS should be able to infer the return type OK for this function:

```suggestion
export function parseEventFromSubject(value: string) {
```